src/components/VertDismissCard.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { useWishlist } from "../context/wishlist-context";
+
+const VertDismissCard = ({product}) => {
+    const {wishlistDispatch} = useWishlist();
+    return (
+        <div className="card-badge" id={product._id}>
+            <div className="product">
+                <div className="prod-container">
+                    <img src={product.img} alt="product" />
+                    <span className="btn-badge-dismiss" onClick={() => wishlistDispatch({type: 'REMOVE_FROM_WISHLIST', payload: product})}><i className="fa-solid fa-heart px-sm" id="close-card" style={{color: '#f34e4e'}}></i></span> 
+                </div>
+                <div className="prod-content">
+                    <h5>{product.title}</h5>
+                    <div className="prod-info">
+                        <span className="prod-price">Rs {product.price}</span>
+                        <span className="ratings-con" style={{fontSize: '1rem'}}>
+                            <i className="fas fa-star"></i>{product.rating}
```

Comment:
Can destructure product properties once so you won't have to write product repeatedly